content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+
+### Passwords and two-factor authentication
+
+*   Phishing and spam emails may increase and several parties my try to extort you. Cyber criminals may distribute and sell your information to other criminals and some may try guessing your passwords on different services. Make sure you have a strong password in all services and start using two-factor authentication if possible. You should also change the password of the email account which has been leaked.
+*   Instructions for creating strong passwords
+    1. The longer your password is, the safer it is. Good minimum length is 16 characters.
+    2. Use both small and capital letters combined with special and numeric characters.
+    3. Typos, dialects, spoken idioms and other "broken words" make your password stronger.
+    4. Use a unique password for every service.
+    5. Be especially careful with important passwords, like the email account passwort that can be used to reset other passwords.
+    6. Never reveal your passwords to anyone! Official authorities will never ask you to disclose your password!
+    7. Start using a password manager software so you won't need to remember your passwords. For example, the Finnish [F-Secure ID Protection](https://www.f-secure.com/fi/home/products/id-protection) or open source [KeePassX](https://www.keepassx.org) are solid programs for this.
+    *   [More information on password security from the Finnish National Cyber Security Centre](https://www.kyberturvallisuuskeskus.fi/en/ncsc-news/instructions-and-guides/longer-better-how-create-strong-password)
+*   Start using two-factor authentication on all services, such as email or social media sites. Two-factor authentication is usually mandatory in banking services. Two-factor authentication confirms your identity using two separate methods. While SMS is possible, a special authentication app is recommended, such as Google Authenticator ([Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2),[iOS](https://apps.apple.com/fi/app/google-authenticator/id388497605)) or [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator).
+
+### Check your insurance policy
+
+*   Many home insurances provide limited legal assistance for victims of an identity fraud. The data privacy legislation may give you a right for damage compensation.
+
+### Make a request to access your personal data from Vastaamo
+
+*   According to European General Data Protection Regulation (GDPR), you have the right to know what data Vastaamo has collected about you. This can help you decide what you want to do next.
+*   Contact Vastaamo with their [request for access to personal data](https://vastaamo.fi/files/rekisteritietojen_tarkastuspyynto.pdf) -form. More information about Vastaamo Data Protection is available from [their website](https://vastaamo.fi/tietosuoja/). (currently only in Finnish)
+*   Understand that it may take several weeks to get an answer.
```

Comment:
It may take up to several weeks to obtain an answer. However, by EU Law, the company must comply with a request for information